server/index.ts

```diff
@@ -11,16 +13,43 @@ const { PORT = 3000 } = process.env;
 const CLIENT = path.resolve(__dirname, '..', 'dist');
 
 const app = express();
+const socket = express();
+const server = createServer(socket);
+const io = new Server(server, {
+  connectionStateRecovery: {},
+  cors: {
+    origin: 'http://localhost:3000',
+    methods:['GET','POST']
+  }
+})
 
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(express.static(CLIENT));
 app.use('/api', routes);
 
-app.get('*', (req: Request, res: Response) => {
+app.get('/', (req: Request, res: Response) => {
```

Comment:
I made this change to make the websocket connection work. I didn't see anyting broken by it, but let me know if it's a problem. I would ideally like to keep it as '/', it makes more semantic sense to me.